Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -6,8 +6,8 @@ import HeaderContent from './HeaderContent';
 
 function HeaderContainer() {
   return (
-    <div className="HeaderContainer">
-      <ImageThumbnail/>
+    <div className="headerContainer">
```

Comment:
For class names we don't usually use camelCase. Using a dash is the convention option (`header-container`) 👍